server/api/versions/index.get.ts

```diff
@@ -0,0 +1,28 @@
+import { fetchRApi } from "@/server/utils/rApi";
+import type { RApiResponse } from "@/server/utils/rApi";
+import packageJson from "@/package.json";
+
+const rApiVersionEndpoint = "/";
+
+interface VersionDataResponse extends RApiResponse {
+  data: {
+    "daedalus": string
+    "daedalus.api": string
+  } | null
+}
+
+interface SuccessResponse {
+  daedalusModel: string
+  daedalusApi: string
+  daedalusWebApp: string
+}
```

Comment:
It feels like this sort of api model type is going to be useful to the client, so it might be better to export it with a less generic name, and to define it in a `models` or `types` folder perhaps.